apps/frontend/src/pages/MainPage.tsx

```diff
@@ -16,6 +16,7 @@ import {
 } from '../context/ModalContext';
 import { Modals } from '../components/Modals/Modals';
 import { BottomBar } from '../components/BottomBar';
+import { Map } from '../components/Map/Map';
```

Comment:
This seems unused

Reply:
Jepp – I'll add some simple linting CI to ensure stuff like this is picked up automatically 😅